templates/mitaka/neutron_ovs_agent.conf

```diff
@@ -15,7 +15,8 @@ rabbit_virtual_host = {{ rabbit_virtual_host }}
 rabbit_ha_queues = {{ rabbit_ha_queues }}
 
 [securitygroup]
-firewall_driver = neutron.agent.firewall.NoopFirewallDriver
+firewall_driver = neutron.plugins.hyperv.agent.security_groups_driver.HyperVSecurityGroupsDriver
```

Comment:
Yep. :) But I will change it to native OVS firewall driver once that it production-ready. That would happen in the next OVS release (2.7).